core/src/main/java/apoc/export/json/JsonImporter.java

```diff
@@ -130,11 +141,28 @@ private void manageNode(Map<String, Object> param) {
             lastLabels = labels;
         }
         if (!labels.equals(lastLabels)) {
+            checkConstraints(labels);
             flush();
             lastLabels = labels;
         }
     }
 
+    private void checkConstraints(List<String> labels) {
+        if (labels.isEmpty()) {
+            return;
+        }
+        try (final Transaction tx = db.beginTx()) {
+            final Schema schema = tx.schema();
+            final String importIdName = importJsonConfig.getImportIdName();
```

Comment:
what does this config do? We looked at the doc, but the description isn't super clear. https://neo4j.com/labs/apoc/4.2/overview/apoc.import/apoc.import.json/#_config_parameters

Reply:
This is the name of the property to be populated with the "id" field present into the json. 
For example `{"type":"node", "labels":["Language"], "id":"10"}`, with importIdName:`foo`, will create a `(:User {foo: "10"})`.
(I changed the documentation as well)